app/scripts/controllers/swaps.js

```diff
@@ -61,6 +67,7 @@ const initialState = {
     topAggId: null,
     routeState: '',
     swapsFeatureIsLive: false,
+    swapsQuoteRefreshTime: 0,
```

Comment:
Nit: This value is never used, but it might be better to set `FALLBACK_QUOTE_REFRESH_TIME` as the default value here anyway so that it's more clear that it's never going to be zero. A quote refresh time of zero would be less than ideal.